sharktank/sharktank/layers/configs/llm_configs.py

```diff
@@ -633,9 +633,6 @@ def to_properties(self) -> "PropertyValueType":
         res["activation_dtype"] = dtype_to_serialized_name(self.activation_dtype)
         res["attention_dtype"] = dtype_to_serialized_name(self.attention_dtype)
         res["fake_quant"] = self.fake_quant
-        res["tensor_parallelism_size"] = self.tensor_parallelism_size
```

Comment:
`tensor_parallelism_size` is not being saved by parallelism_config

Reply:
Nevermind, its not that.